mautic/mautic
# Add seconds unit for Visits specific URL Point action

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N ]
| New feature/enhancement? (use the a.x branch)      | [ Y ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ N ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR adds seconds unit in Visits specific URL Point action
![image](https://github.com/mautic/mautic/assets/8580942/d6695aeb-ad2b-4647-ba6b-987e5b8a39fc)

Please note that the contact tracking on external websites may not work properly on 5.x, see #12325 for more details.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
3. Add a new Point Action 
4. Choose **Visits Specific URL** from **When a contact...** select
5. Set the Page URL
6. Set Total time spent to e.g. 10 seconds and Save Point action
7. Add Mautic tracking code to your external website
8. Add website to valid domains in Mautic CORS configuration
9. Visit the URL that you defined in Point action
10. Go to any other subpage that still has the Mautic tracking code (this is how Mautic determinates the time spent on page)
11. Go back to the URL defined in the point action to trigger the page visit - this step should not be necessary to trigger this event, but it seems like it work this way right now (I have described this in issue #12336)
12. Find the contact and check if the points were added
